libbpf-tools/biolatency.bpf.c

```diff
@@ -52,17 +59,31 @@ int trace_rq_start(struct request *rq)
 }
 
 SEC("tp_btf/block_rq_insert")
-int BPF_PROG(block_rq_insert, struct request_queue *q, struct request *rq)
+int block_rq_insert(u64 *ctx)
 {
-	return trace_rq_start(rq);
+	/**
+	 * commit a54895fa (v5.11-rc1) changed tracepoint argument list
```

Comment:
we already landed PR with these changes. Please rebase.

Reply:
fixed